### PR TITLE
Newsletter-277: translate into Chinese

### PR DESCRIPTION
**Pull request by hulatown**

### CODE REVIEW

File: _posts/zh/newsletters/2023-11-15-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+- **<!--eliminating-malleability-from-ephemeral-anchor-spends-->从临时锚点花费中消除熔融性：** Gregory Sanders 在 Delving Bitcoin 论坛上[发表了][sanders mal]一个对[临时锚点][topic ephemeral anchors]提案的微调。该提案将允许交易包括一个零价值输出，其输出脚本为任何人都可以花费。由于任何人都可以花费输出，因此任何人都可以使用 [CPFP][topic cpfp] 来对创建这个输出的交易进行费用提升。这对于诸如 LN 等多方合同协议非常方便，因为交易通常会在能够准确预测应支付的费率之前签署。临时锚定点允许合约任何一方添加他们认为必要的费用。如果任何其他一方，或者任何其他用户出于任何原因，想要添加更高费用，则他们可以用自己更高费率的 CPFP 费用提升来[替代][topic rbf]之前的 CPFP 费用提升。
+
+    提议使用的任何人都可以花费的脚本类型是一个输出脚本，它由等效的 `OP_TRUE` 组成，可以通过具有空输入脚本的输入进行花费。正如Sanders 本周发布的那样，使用传统输出脚本意味着花费它的子交易具有可熔铸变型的交易索引号（txid），例如，任何矿工都可以向输入脚本添加数据以更改子交易的 txid。这可以使得除了用于费用增加之外，将该子交易用于任何其他用途都是不明智的，因为即使子交易得到确认，也可能以不同的 txid 确认，从而使任何孙交易无效。
+
+    Sanders 建议改用原先为后来隔离见证（Segwit）升级所保留的输出脚本之一。这会使用略多的区块空间————SegWit 使用四个字节，而裸的 `OP_TRUE` 仅使用一个字节，但可以消除关于交易熔融性的任何顾虑。在该主题上进行了一些讨论后，Sanders 随后提出同时提供两种版本：一个 `OP_TRUE` 版本，适用于不关心熔融性且希望尽量减小交易大小的人，以及一个 SegWit 版本，略大但不允许子交易被变异。此讨论主题的进一步探讨侧重于为 SegWit 版本选择额外字节以创建一个易于记忆的 [bech32m 地址][topic bech32]。{% assign timestamp="1:54" %}
+
+## 田野调查：一段 Miniscript 旅程
+
+{% include articles/wizardsardine-miniscript.md extrah="#" %} {% assign timestamp="20:17" %}
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2023-11-15-newsletter.md:22]:
```suggestion
{% include articles/zh/wizardsardine-miniscript.md extrah="#" %}
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]
Few suggestions below.

Additionally can you create a blog for the miniscript article (in the /_posts/zh/ folder)? See how it was done for the FR translation here:
https://github.com/bitcoinops/bitcoinops.github.io/pull/1399/files

File: _posts/zh/newsletters/2023-11-15-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+---
+本周的周报介绍了对临时锚点提案的更新，并附上了来自 Wizardsardine 开发人员关于 miniScript 的贡献性工作报告。还包括我们的常规部分：新软件版本和候选版本的公告，以及对热门比特币基础设施项目的重大变更介绍。
+
+## 新闻
+
+- **<!--eliminating-malleability-from-ephemeral-anchor-spends-->从临时锚点花费中消除熔融性：** Gregory Sanders 在 Delving Bitcoin 论坛上[发表了][sanders mal]一个对[临时锚点][topic ephemeral anchors]提案的微调。该提案将允许交易包括一个零价值输出，其输出脚本为任何人都可以花费。由于任何人都可以花费输出，因此任何人都可以使用 [CPFP][topic cpfp] 来对创建这个输出的交易进行费用提升。这对于诸如 LN 等多方合同协议非常方便，因为交易通常会在能够准确预测应支付的费率之前签署。临时锚定点允许合约任何一方添加他们认为必要的费用。如果任何其他一方，或者任何其他用户出于任何原因，想要添加更高费用，则他们可以用自己更高费率的 CPFP 费用提升来[替代][topic rbf]之前的 CPFP 费用提升。
+
+    提议使用的任何人都可以花费的脚本类型是一个输出脚本，它由等效的 `OP_TRUE` 组成，可以通过具有空输入脚本的输入进行花费。正如Sanders 本周发布的那样，使用传统输出脚本意味着花费它的子交易具有可熔铸变型的交易索引号（txid），例如，任何矿工都可以向输入脚本添加数据以更改子交易的 txid。这可以使得除了用于费用增加之外，将该子交易用于任何其他用途都是不明智的，因为即使子交易得到确认，也可能以不同的 txid 确认，从而使任何孙交易无效。
+
+    Sanders 建议改用原先为后来隔离见证（Segwit）升级所保留的输出脚本之一。这会使用略多的区块空间————SegWit 使用四个字节，而裸的 `OP_TRUE` 仅使用一个字节，但可以消除关于交易熔融性的任何顾虑。在该主题上进行了一些讨论后，Sanders 随后提出同时提供两种版本：一个 `OP_TRUE` 版本，适用于不关心熔融性且希望尽量减小交易大小的人，以及一个 SegWit 版本，略大但不允许子交易被变异。此讨论主题的进一步探讨侧重于为 SegWit 版本选择额外字节以创建一个易于记忆的 [bech32m 地址][topic bech32]。{% assign timestamp="1:54" %}
```
[the remark this answers is not in the document]
Comment (hulatown) [reply _posts/zh/newsletters/2023-11-15-newsletter.md:18]: > @hulatown can you remove the {% assign timestamp="1:54" %} entries for this newsletter?

Sure! Do you mean that we should remove all entries like `{% assign timestamp="<time>" %}` from the newsletter? And the entries don't need to be added in the translated newsletter?

[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/zh/newsletters/2023-11-15-newsletter.md:18]: Correct, they are for the podcast timestamping feature and not applicable to translations at this time.

File: _posts/zh/newsletters/2023-11-15-newsletter.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ lang: zh
 
 ## 田野调查：一段 Miniscript 旅程
 
-{% include articles/wizardsardine-miniscript.md extrah="#" %} {% assign timestamp="20:17" %}
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2023-11-15-newsletter.md]: Looks right, can you please also remove throughout? Thank you @hulatown !

File: _posts/zh/newsletters/2023-11-15-newsletter.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hulatown) [reply _posts/zh/newsletters/2023-11-15-newsletter.md]: Fixed! And will notice in the future

File: _posts/zh/newsletters/2023-11-15-newsletter.md (hunk 3 above)
Comment (hulatown) [reply _posts/zh/newsletters/2023-11-15-newsletter.md]: Done

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Review by bitschmidty — APPROVED** [review]
ACK, thank you for putting up with my feedback @hulatown !

**Pull request merged**